lib/git-fastclone.rb

```diff
@@ -252,9 +241,12 @@ def update_submodules(pwd, url)
 
       threads = []
       submodule_url_list = []
+      output = ''
+      Dir.chdir(File.join(abs_clone_path, pwd).to_s) {
```

Comment:
NIT: Given the prevalence of wanting to run these commands in a different directory, worth adding a `chdir:` option into `fail_on_error` and `fail_pipe_on_error`? Alternatively. we could change the directory someplace further up for more of the execution, but that'd require more thinking about what might break.

Reply:
i think the overall effect (adding chdir: option) is the same? I think I am more concerned about if this is the right code to use: doing this should ensure chdir only happened within this context but i also see warning from ruby about `finding chidir inside another chidr block`) -> Is there a better way than we what we have right now?